<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimelineVisualisationWidget</class>
 <widget class="QMainWindow" name="TimelineVisualisationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayoutCentral">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitterLeftRight">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitterControlsTopBottom">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QDockWidget" name="dockCorpusExplorer">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>3</verstretch>
         </sizepolicy>
        </property>
        <property name="features">
         <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
        </property>
        <property name="windowTitle">
         <string>Corpus Items</string>
        </property>
        <widget class="QWidget" name="dockWidgetCorpusExplorer">
         <layout class="QGridLayout" name="gridLayoutTopTab">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
       <widget class="QDockWidget" name="dockLevelAttributes">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="features">
         <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
        </property>
        <property name="windowTitle">
         <string>Select Levels and Attributes</string>
        </property>
        <widget class="QWidget" name="dockWidgetLevelsAttributes">
         <layout class="QGridLayout" name="gridLayoutBottomTab">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitterCanvasTopBottom">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>6</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QDockWidget" name="dockWidgetVisualisation">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>6</horstretch>
          <verstretch>3</verstretch>
         </sizepolicy>
        </property>
        <property name="windowTitle">
         <string>Visualisation</string>
        </property>
        <widget class="QWidget" name="dockWidgetVisualisationEditorContents">
         <layout class="QGridLayout" name="gridLayoutVisualisationEditor">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
       <widget class="QDockWidget" name="dockWidgetAnnotationEditor">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>6</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="windowTitle">
         <string>Annotation Editor</string>
        </property>
        <widget class="QWidget" name="dockWidgetAnnotationEditorContents">
         <layout class="QGridLayout" name="gridLayoutAnnotationEditor">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
